View Controllers/ProfileViewController.m

```diff
@@ -119,6 +122,11 @@ - (void)viewDidLoad {
     
     // set UI styles
     [self initUIStyles];
+    
+    // set up refresh control
```

Comment:
I'm not going to make you take this out, but this (adding the refresh control) is a good example of a change you'd make in its own branch